experimental/addons/hive-operator/README.md

```diff
@@ -0,0 +1,175 @@
+--set consoleService.type=NodePort,consoleService.nodePort=null \
+--repo https://charts.min.io/ minio
+```
+
+**PostgresSQL**
+
+```shell
+helm install postgresql \
+--version=12.1.5 \
+--namespace prod \
+--set postgresqlUsername=hive \
+--set postgresqlPassword=hive \
+--set postgresqlDatabase=hive \
+--repo https://charts.bitnami.com/bitnami postgresql
+```
+
+## Use
+
+After going through the Installation section and having installed all the dependencies, you will now deploy a Hive metastore cluster and it’s dependencies. Afterwards you can verify that it works.
+
+In order to connect Hive to MinIO we need to create several dependent components like S3Connection, Secret and a SecretClass
+
+**s3-connection**
+
+An S3Connection to connect to MinIO, Apply below YAML:
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: s3-connection-sample
+spec:
+  components:
+    - type: s3-connection
+      name: minio
+      properties:
+        host: minio
+        port: 9000
+        accessStyle: Path
+        credentials:
+          secretClass: hive-s3-secret-class
+```
+
+**secret**
+
+Credentials for the S3Connection to log into MinIO, Apply below YAML:
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: hive-secret-sample
+spec:
+  components:
+    - type: k8s-objects
+      name: k8s-demo-secret
+      properties:
+        objects:
+          - apiVersion: v1
+            kind: Secret
+            metadata:
+              name: hive-s3-secret
+              labels:
+                secrets.stackable.tech/class: hive-s3-secret-class
+            stringData:
+              accessKey: hive
+              secretKey: hivehive
+```
+
+**secret-class**
+
+A SecretClass for the credentials to the Minio. The credentials were defined in the installation of Minio via helm, Apply below YAML:
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: secret-class-sample
+spec:
+  components:
+    - type: secret-class
+      name: hive-s3-secret-class
+      properties:
+        backend:
+          k8sSearch:
+            searchNamespace:
+              pod: {}
+```
+
+**hive-cluster**
+
+And lastly the actual Apache Hive cluster definition, Apply below YAML:
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: hive-postgres-cluster
+spec:
+  components:
+    - type: hive-cluster
+      name: hive-postgres-cluster
+      properties:
+        image:
+          productVersion: 3.1.3
+          stackableVersion: 23.1.0
+        clusterConfig:
+          database:
+            connString: jdbc:postgresql://postgresql:5432/hive
+            user: hive
+            password: hive
+            dbType: postgres
+          s3:
+            reference: minio
+        metastore:
+          roleGroups:
+            default:
+              replicas: 1
+```
+
+Verify that it works
+
+```shell
+$ kubectl get statefulset -n prod
```

Comment:
can we use vela status to check?

Reply:
Can you try `vela status something --tree --detail`?